src/components/react_table.js

```diff
@@ -78,7 +84,7 @@ function Table(props) {
             {allColumns.map(column => (
               <div key={column.id}>
                 <label>
-                  <input type="checkbox" {...column.getToggleHiddenProps()} />{' '}
+                  <input type="checkbox" {...column.getToggleHiddenProps()}  onClick={(e) => handleClick(e, column)} />{' '}
```

Comment:
Override the existing `onClick` function, which will pass the selected column we want to filter out to the event handler.